### DISCUSSION TITLE
Python generators example

### DISCUSSION THREAD

**Discussion by sopheya** [community/community#115500] [Programming Help] [answered]
### Body

Can someone give me a good example of a useful Python generator? 

Can anyone write an ROT13 function using a generator to help me understand?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by nickkipshidze** [comment, marked as answer]
Here's a simple ROT13 implementation in Python then:
```python
def rot13(string):
    return [chr(ord(c) + 13) for c in string]
```
I used `c` variable as an short for `character`. Tell me if it helped.

**Comment by elbekgiyozov** [comment]
def rot13(string):
    return [chr(ord(c) + 13) for c in string]
    
    (please mark as answer 🙏🙏🙏)
